python/shotgun_model/shotgun_standard_item.py

```diff
@@ -8,16 +8,15 @@
 # agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
-from .util import sanitize_qt, get_sg_data
 
 from tank.platform.qt import QtGui
 
+from .util import sanitize_qt
+from .util import get_sg_data as util_get_sg_data
 
 class ShotgunStandardItem(QtGui.QStandardItem):
     """
     Special implementation of StandardItem which bridges PyQt and PySide.
-
-    Do not construct this object directly - instead use the ShotgunModel.create_item() method.
```

Comment:
i *think* the reason for this comment is because `create_item` will add the item to the internal data structures that keep references to everything to prevent memory leaks, double frees etc. Otherwise you can create a crash by creating one of these items directly then plug it into a model and there will be issues with GC. But I am not 100% sure this is the case. It may be worth adding some further comments here to provide more details. At least in earlier versions of pyside, we found that the MVC stuff was very brittle in subtle ways, hence all these methods and helpers..